packages/clarity-js/src/data/metadata.ts

```diff
@@ -129,11 +142,12 @@ function num(string: string, base: number = 10): number {
 }
 
 function user(): string {
-  let id = cookie(Constant.CookieKey);
-  return id && id.length > 0 ? id : shortid();
+  let id = getCookie(Constant.CookieKey);
+  // Splitting and looking up first part for forward compatibility, in case we wish to store additional information in a cookie
+  return id && id.length > 0 ? id.split(Constant.Separator)[0] : shortid(); 
```

Comment:
>Separator [](start = 49, length = 9)

CookieSeparator to make it clear what is a separator for (shortened like CookieSep or something is fine).

Reply:
I will change it to Pipe which is what the character is.

Reply:
Done